examples/typescript/any_authenticator_account_abstraction.ts

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-console */
```

Comment:
We need a similar file for permissioned delegation to make sure your sdk for permission delegation also works.